src/drive/web/modules/navigation/Index.spec.js

```diff
@@ -1,3 +1,7 @@
+jest.mock('cozy-keys-lib', () => ({
+  withVaultClient: jest.fn().mockReturnValue({}),
+  useVaultClient: jest.fn()
+}))
```

Comment:
Can we keep the jest.mock below the import/require 🙏 ?

Reply:
Sure!